### ISSUE TITLE
having some trouble with Python modules that won't load

### ISSUE THREAD

**Issue by createcandle** [WebThingsIO/addon-list#1287]
When intalling Voco on the Raspberry Pi Zero, I noticed that it hadn't loaded the [Rapidfuzz](https://pypi.org/project/rapidfuzz/) and [ALSA library](https://pypi.org/project/pyalsaaudio/) that it depends on.

```
No module named 'rapidfuzz.cpp_impl'
No module named 'alsaaudio'
```

I checked the `lib` folder, and I can see this form an `ls`:
```
rapidfuzz
rapidfuzz-0.14.2-py3.5.egg-info
pyalsaaudio-0.9.0-py3.5.egg-info 
```
(but no actual `alsaaudio` folder?)

So I thought perhaps the `path` variable is the issue. Then again, other modules load fine.

Is this still the correct thing to put at the top of a python addon?

```
import os
#from os import path
import sys
sys.path.append(os.path.join(os.path.dirname(os.path.abspath(__file__)), 'lib'))
```

Or could it be because radidfuzz relies on C, and isn't compatible with a 'no binaries' approach?

**Comment by createcandle** [comment]
I've created a pull request for Voco that removes the Rapidfuzz dependency. That's one way to solve the issue :-)

One to go..

**Comment by mrstegeman** [comment]
Looks to me like it's there (notice the first entry):

```
pi@gateway:~/voco $ tar tzf voco-2.5.6.tgz | grep '^package/lib/' | sort | grep alsa
package/lib/alsaaudio.cpython-37m-arm-linux-gnueabihf.so
package/lib/pyalsaaudio-0.9.0.egg-info/
package/lib/pyalsaaudio-0.9.0.egg-info/PKG-INFO
package/lib/pyalsaaudio-0.9.0.egg-info/SOURCES.txt
package/lib/pyalsaaudio-0.9.0.egg-info/dependency_links.txt
package/lib/pyalsaaudio-0.9.0.egg-info/installed-files.txt
package/lib/pyalsaaudio-0.9.0.egg-info/top_level.txt
```

**Comment by createcandle** [comment]
That's some impressive piping.

TLDR: I'm removing this dependency too.


I don't see any `alsaaudio` directory though - assuming that's what an import command is looking for an `init` file in? The list has no actual python files? Is that first `.so` file really all that's required?

I tried a similar command in the `lib` folder on the Pi Zero where Voco was just updated:

`ls -R | sort | grep alsa`

shows

```
./pyalsaaudio-0.9.0-py3.5.egg-info:
Choibalsan
alsaaudio.cpython-35m-arm-linux-gnueabihf.so
pyalsaaudio-0.9.0-py3.5.egg-info

```
I also checked the logs, and the Alsa import error is still there. 

One thing I noticed is that it's the only import with a simple `import module_name` structure. The other imports all follow `from module_name import X`. Perhaps that's it.

Either way, while diving deeper into my code, I realised that this dependency could also be removed.
